gov2/iam/README.md

```diff
@@ -2,266 +2,93 @@
-
-The unit test accepts a similar value in _config.json_.
-
-### ListAccessKeys/ListAccessKeysv2.go
-
-This example retrieves the access keys for your IAM account.
-
-`go run ListAccessKeysv2.go -u USER-NAME -m MAX-KEYS`
-
-- _USER-NAME_ is the name of the user for which the keys are listed.
-- _MAX-KEYS_ is the maximum number of keys to display.
-  If this value is negative, the code example sets it to 10. 
-
-The unit test accepts similar values in _config.json_.
-
-### ListAccountAliases/ListAccountAliasesv2.go
-
-This example retrieves the aliases for your IAM account.
-
-`go run ListAccountAliasesv2.go [-m MAX-ITEMS]`
-
-- _MAX-ITEMS_ is the maximum number of aliases to show.
-  If this value is less than zero, the code example sets it to 10.
-  
-The unit test accepts a similar value in _config.json_.
-
-### ListAdmins/ListAdminsv2.go
-
-This example lists the number IAM users and those who have administrative privileges.
-
-`go run ListAdminsv2.go [-d]`
-
-- **-d** to list the user and administrator names.
-
-The unit test accepts a similar value in _config.json_.
-
-### ListServerCerts/ListServerCertsv2.go
-
-This example retrieves the server certificates.
-
-`go run ListServerCertsv2.go`
-
-### ListUsers/ListUsersv2.go
-
-This example retrieves a list of your IAM users.
-
-`go run ListUsersv2.go [-m MAX-USERS]`
-
-- _MAX-USERS_ is the maximum number of users to list.
-  The code example restricts this to the range of 0 to 100.
-  The default value is 10.
-  
-### UpdateAccessKey/UpdateAccessKeyv2.go
-
-This example sets the status of an IAM access key to active.
-
-`go run UpdateAccessKeyv2.go -k KeyID -u USER-NAME`
-
-- _KEYID_ is the access key to activate.
-- _USER-NAME_ is the name of the user activating the key.
+## Prerequisites
 
-The unit test accepts similar values in _config.json_.
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the AWS SDK for Go Developer Guide.
 
-### UpdateServerCert/UpdateServerCertv2.go
 
-This example renames an IAM server certificate.
 
-`go run UpdateServerCert/UpdateServerCertv2.go -c CERTIFICATE-NAME -n NEW-NAME`
+## ⚠ Important
 
-- _CERTIFICATE-NAME_ is the original name of the server certificate.
-- _NEW-NAME_ is the new name of the server certificate.
+- As an AWS best practice, grant this code least privilege, or only the 
+  permissions required to perform a task. For more information, see 
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) 
+  in the *AWS Identity and Access Management 
+  User Guide*.
+- This code has not been tested in all AWS Regions. Some AWS services are 
+  available only in specific Regions. For more information, see the 
+  [AWS Region Table](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/)
```

Comment:
Update to the "For more information..." sentence: 
For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) on the AWS website.